### ISSUE TITLE
restore-exif-mtime: Still does not run

### ISSUE THREAD

**Issue by RichiH** [glasserc/etc#4]
```
% restore-exif-mtime
Traceback (most recent call last):
  File "/home/richih/bin/restore-exif-mtime", line 38, in <module>
    walk_dirtree('.')
  File "/home/richih/bin/restore-exif-mtime", line 32, in walk_dirtree
    realtime = get_time_exif(fname)
  File "/home/richih/bin/restore-exif-mtime", line 15, in get_time_exif
    tags = EXIF.process_file(f)
NameError: global name 'EXIF' is not defined
% echo $?
1
%
```

**Comment by glasserc** [comment]
Oops! Fixed in f5ab0e0e5af7d63754443d9d1fbbd2156151cf51. Can you try again?

**Comment by RichiH** [comment]
Works, thanks.
